### PR TITLE
Release notes for 16.10 and 16.11 (so far)

### PR DESCRIPTION
**Pull request by rainersigwald**
As part of catching up after my parental leave, I looked through change history, so I wrote up these notes.

### CODE REVIEW

**Review by jeffkl — APPROVED** [review]

File: Changelog.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# MSBuild 16.10.0
+
+This version of MSBuild will with Visual Studio 2019 version 16.10.0 and .NET SDK 5.0.300.
```
Comment (akoeplinger) [inline Changelog.md:3]:
```suggestion
This version of MSBuild will ship with Visual Studio 2019 version 16.10.0 and .NET SDK 5.0.300.
```

File: Changelog.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# MSBuild 16.10.0
```
Comment (akoeplinger) [inline Changelog.md:1]: do you intend to backfill the changelogs for earlier releases from https://github.com/dotnet/msbuild/releases ?

**Comment by KirillOsenkov** [comment]
Can these release notes include a reference to itself? "These release notes were published"

j/k ;)

File: Changelog.md (hunk 2 above)
Comment (akoeplinger) [reply Changelog.md:1]: Hm I don't think I've ever seen a project where a Changelog.md only contained the current version.

Most have all of the releases in it and a section for v.current, e.g. https://github.com/emscripten-core/emscripten/blob/main/ChangeLog.md

I'd prefer that, or at least have links to the earlier release notes in the doc.

**Review by BenVillalobos — APPROVED** [review]

File: Changelog.md (hunk 2 above)
Comment (rainersigwald) [reply Changelog.md:1]: Sounds reasonable to me. I'll add at least the 16.9 release.

File: Changelog.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+* Projects can now specify `AdditionalTargetFrameworkInfoProperty` items to indicate that referencing projects should get those properties exposed as `AdditionalPropertiesFromProject` metadata on resolved reference items. (#5994).
+* The `Unzip` task now accepts `Include` and `Exclude` arguments to filter what is extracted from the zip file (#6018). Thanks, @IvanLieckens!
+* The `-graph:noBuild` command line argument can be used to validate that a graph is buildable without actually building it (#6016).
+* When building a solution filter, `$(SolutionFilterName)` is now defined (#6171).
+* `TaskParameterEventArgs` allow logging task parameters and values in a compact, structured way (#6155). Thanks, @KirillOsenkov!
+* ClickOnce publish now supports Ready To Run (#6244).
+* .NET 5.0 applications may now specify a toolset configuration file (#6220).
+* `ResolveAssemblyReferences` can now consume information about assemblies distributed as part of the SDK (#6017).
+* Allow constructing a `ProjectInstance` from a `ProjectLink` (#6262).
+* Introduce cross-process resource management for tasks (#5859).
```
Comment (Forgind) [inline Changelog.md:25]: SolutionFilterName? Or still too up in the air? (I haven't seen a response on that, by the way.)

File: Changelog.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
+# MSBuild 16.10.0
+
+This version of MSBuild will ship with Visual Studio 2019 version 16.10.0 and .NET SDK 5.0.300.
+
+## What's new
+
+* MSBuild now targets .NET 5.0 and .NET Framework 4.7.2.
+* MSBuild is faster and uses less memory.
+* Binary logs are smaller and have less performance overhead.
```
Comment (Forgind) [inline Changelog.md:9]: master -> main rename?

**Review by Forgind — APPROVED** [review]
I only combed through the first 2.5 pages of PRs, so I'm sure I missed things, but I don't think it's worth it for me to keep going.

File: Changelog.md (hunk 3 above)
Comment (rainersigwald) [reply Changelog.md:27]: https://github.com/dotnet/msbuild/pull/6351/files#diff-c275fec9c453c2a42515bc5ab47e30fa4130ff1426aef2f6e000a9a34e122cb8R19 ?

**Review by cdmihai — APPROVED** [review]

**Pull request merged**